### PR TITLE
format the output model input

### PR DESCRIPTION
**Pull request by zpcore**
Fix https://github.com/pytorch/xla/issues/6864

### CODE REVIEW

**Comment by ysiraichi** [comment]
Does this work for all the models in the issue? I'd imagine we'd have to do something similar to [their old implementation](https://github.com/pytorch/benchmark/blob/43866040439cb9a2d1c5508c72609dce7250fb2d/torchbenchmark/util/framework/huggingface/model_factory.py#L140-L148)

File: benchmarks/torchbench_model.py [hunk 1]
```diff
@@ -264,7 +264,9 @@ def set_up(self):
     benchmark = self.load_benchmark()
 
     self.module, self.example_inputs = benchmark.get_module()
-
+    if isinstance(self.example_inputs,
```
Comment (ysiraichi) [inline benchmarks/torchbench_model.py:267]: Leave a comment here, mentioning the pytorch/benchmark PR. It would be good to know in the future.

**Comment by ysiraichi** [comment]
I have a few more questions:

1. Would this also work with older versions of pytorch/benchmark? 
2. What's the policy when a breaking change is merged on pytorch/benchmark's main branch? Do we follow:
    - the main branch
    - the pinned commit found in the main branch of pytorch/pytorch
    - some other pinned commit

**Comment by zpcore** [comment]
Thanks for pointing this out!

> Would this also work with older versions of pytorch/benchmark?

It should work. Currently only the hf models inputs have been changed into the dict type.

> What's the policy when a breaking change is merged on pytorch/benchmark's main branch?

I don't have a good solution at this moment, but I expect the changes like regarding the input type should be rare. I prefer sync up to the latest pytorch/benchmark repo at this moment. For the long term, once your proposal of merging the pytorch/xla/benchmarks is landed and implemented, we should get rid of this issue with their CI.

**Comment by zpcore** [comment]
Hello, pin to see if someone can make an approval.

**Review by vanbasten23 — APPROVED** [review]
Thanks for the fix!

**Comment by zpcore** [comment]
Hi @ysiraichi , I will merge the change. You may need to update the python/benchmark PIN to the head when you run torchbench or error may occur for those HF models.

**Pull request merged**